io.github.revisto.DrumMachine.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id" : "io.github.revisto.DrumMachine",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "drum-machine",
+    "finish-args" : [
+        "--share=network",
```

Comment:
does it really need network access?

Reply:
No, I removed that. It was just added by default from GNOME Builder template, but the app doesn't need any network access to function.